packages/opal-server/opal_server/server.py

```diff
@@ -294,6 +295,8 @@ async def startup_event():
             try:
                 await self.start()
                 self._task = asyncio.create_task(self.start_server_background_tasks())
+                #start pulling tasks
+                DataUpdatePublisher.create_polling_updates(self.publisher, opal_server_config.DATA_CONFIG_SOURCES)
```

Comment:
I think the return value from `create_polling_updates` needs to be kept somewhere so it won't be GCed.